config/application.rb

```diff
@@ -24,6 +25,13 @@ class Application < Rails::Application
 
     AppConfig.setup(configuration)
 
+    root_config = configuration.except(['development', 'production', 'test'])
```

Comment:
don't love this, but works right now and behavior probably should end up going in `AppConfigReader` rather than here or in `IdentityConfig`

Reply:
+1 definitely would love to move this out too